Notice that this does quite a good job of drawing a line through our data.  And the data scientist can plug in whatever ad spending budget she wants to find the corresponding prediction.  For example, below we'll see the predicted sales for 5000.

In [69]:
regression.predict(5000)

array([2087.02687877])

Moreover, if she wants to see the general relationship between spending and sales she can do that too.

In [70]:
regression.coef_

array([0.38675261])

In [71]:
regression.intercept_

153.26385079539216

So from the above method calls she can see that every additional dollar spent on advertising leads to an increase of .38 number of T-shirts sold, or in other words it takes about three dollars to sell an additional T-shirt.  The model also tells her that if the advertising budget were zero, the company would sell roughly 153 T-shirts.

### Time for questions

Now at this point we should have a number of questions.  What does `coef_` and `intercept_` mean, and how do they tell her this information?  Do these numbers have anything to do with the orange line above?  And what happens when our linear regression model calls `fit`?  Let's answer each of these questions in the following lessons.  